ios/Exponent/Versioned/Core/Api/EXNotifications.m

```diff
@@ -292,4 +319,8 @@ - (UNMutableNotificationContent *)_localNotificationFromPayload:(NSDictionary *)
   return content;
 }
 
+-(NSString *) prefixWithExperience:(NSString *) identifier {
+  return [NSString stringWithFormat:@"%@%@%@", self.expId, @"@@@", identifier];
```

Comment:
Just use `:` instead of `@@@`. We don't allow `:` in usernames or project names (we validate that project slugs match a regex that doesn't include `:`).